tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = py39, lint, coverage, black
 skipsdist = True
 
 [testenv]
+basepython = python3.9
```

Comment:
This fixed the following linting error:

```
lint run-test: commands[0] | pylint -rn ecosystem tests
Traceback (most recent call last):
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
             ^^^^^^^^^^^^
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/pylint/__init__.py", line 21, in run_pylint
    from pylint.lint import Run as PylintRun
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/pylint/lint/__init__.py", line 76, in <module>
    from pylint.lint.parallel import check_parallel
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/pylint/lint/parallel.py", line 7, in <module>
    from pylint import reporters
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/pylint/reporters/__init__.py", line 26, in <module>
    from pylint import utils
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/pylint/utils/__init__.py", line 46, in <module>
    from pylint.utils.ast_walker import ASTWalker
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/pylint/utils/ast_walker.py", line 6, in <module>
    from astroid import nodes
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/astroid/__init__.py", line 53, in <module>
    from astroid import inference, raw_building
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/astroid/inference.py", line 33, in <module>
    import wrapt
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/wrapt/__init__.py", line 10, in <module>
    from .decorators import (adapter_factory, AdapterFactory, decorator,
  File "/Users/frankharkins/Documents/ecosystem/.tox/lint/lib/python3.11/site-packages/wrapt/decorators.py", line 34, in <module>
    from inspect import ismethod, isclass, formatargspec
ImportError: cannot import name 'formatargspec' from 'inspect' (/usr/local/Cellar/python@3.11/3.11.7_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/inspect.py)
```

Reply:
Another approach could be updating pylint, and if they're pinned, astroid and wrapt.

Reply:
That's a better idea